dockerfiles/zesty/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:precise
+FROM ubuntu:zesty
```

Comment:
Heh, it's already been gone/unsupported for a while now: https://github.com/docker-library/official-images/pull/2908 (Apr 28)

Removing it here won't actually remove the tags -- `neurodebian:precise` will still be available for users to pull, it just won't be listed under "Supported tags" (which also means we won't ever rebuild it, but we won't currently do that anyhow because the base image is no longer supported, and I'm sure Canonical will likely remove it from the mirrors before too long).